src/components/config/parse.ts

```diff
@@ -37,10 +38,16 @@ export const parseConfig = async (
   type: string
 ): Promise<Partial<Config>> => {
   try {
-    const configString = isUrl(source)
+    let configString = isUrl(source)
       ? await fetchConfig(source)
       : readFileSync(source, 'utf-8')
 
+    if (configString.length === 0) {
+      const configFileName = 'monika.yml'
+      await createConfigFile(configFileName)
+      configString = readFileSync(configFileName, 'utf-8')
+    }
+
```

Comment:
Do we need to create the placeholder config file here? This could cause a confusing bug when the file is from remote URL. I think it should show error when the remote file cannot be reached so that the user knows.

We should create the placeholder config file only when:

1. User is not using `-c` flag.
2. And there is no the default monika.yml file in the current directory.

Reply:
agree. removed placeholder from parse.